README.md

```diff
@@ -17,18 +17,30 @@ If running on openshift simply run `make` which will build and push the image to
 
 ## **Usage**
 ### **Usage on openshift platform**
-Once we have OLS deployed and running on openshift cluster, in order to trigger the load test, deploy [assets/ols-load-generator.yaml](https://github.com/vishnuchalla/ols-load-generator/blob/perf-testing/assets/ols-load-generator.yaml) replaced with your corresponding envs values onto the cluster.
+Once we have OLS deployed and running on openshift cluster, in order to trigger the load test follow the below steps.
 
-### **Envs**
+
```

Comment:
@vishnuchalla before we create the secret the - namespace creation is needed. It will be nice to add here .